service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -37,18 +37,23 @@ public static String buildSubjectLine(final CaseData caseData) {
 
     public static String buildSubjectLineWithHearingBookingDateSuffix(final String subjectLine,
                                                                       final List<Element<HearingBooking>>
-                                                                          hearingDetails) {
+                                                                          hearingBookings) {
         String hearingDate = "";
-        if (isNotEmpty(hearingDetails)) {
-            hearingDate = " hearing " + dateFormatterService.formatLocalDateToString(
-                hearingBookingService.getMostUrgentHearingBooking(
-                    hearingDetails).getStartDate().toLocalDate(), FormatStyle.MEDIUM);
+        if (isNotEmpty(hearingBookings)) {
+            hearingDate = buildSubjectLineSuffixText(hearingBookings);
```

Comment:
Why not `buildNextHearingDateLabel` or similar?

Reply:
hmmmm, thought of that. But its not really a label

Reply:
`buildSubjectLine`, thoughts?